design/bower.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "HelloIonic",
```

Comment:
right now we only have single dependence? that is ionic. no external libraries have been used?